debugger/src/main/java/org/codice/acdebugger/impl/DebugContext.java

```diff
@@ -66,7 +67,7 @@ public Backdoor backdoor() {
   }
 
   /**
-   * Accesses system properties util functionality.
+   * Accesses system properties util functionnality.
```

Comment:
✏️ It was right the first time. :)

Reply:
You say that now ... wait until I change it again 😄